obal/data/modules/koji_build.py

```diff
@@ -0,0 +1,69 @@
+"""
+Release a package to koji
+"""
+
+import re
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.koji_wrapper import koji, KojiCommandError # pylint:disable=import-error,no-name-in-module
+from ansible.module_utils.obal import package_exists  # pylint:disable=import-error,no-name-in-module
+
+
+def main():
+    """
+    Build a package in koji
+    """
+    module = AnsibleModule(
+        argument_spec=dict(
+            srpm=dict(type='path', required=True),
+            scratch=dict(type='bool', required=False, default=False),
+            tag=dict(type='str', required=True),
+            nevr=dict(type='str', required=True),
+            package=dict(type='str', required=True),
+            tag_check=dict(type='bool', require=False, default=True),
```

Comment:
```suggestion
            tag_check=dict(type='bool', required=False, default=True),
```

(technically not needed, as r=False is default)